bundles/org.openhab.binding.siemensrds/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,201 @@
+			</channel>
+
+			<channel id="outsideTemperature" typeId="temperature">
+				<label>Outside Temperature</label>
+				<description>Actual outside temperature</description>
+			</channel>
+
+			<channel id="greenLeafScore" typeId="greenLeafScore">
+				<label>Green Leaf Score</label>
+				<description>Energy saving level</description>
+			</channel>
+
+			<channel id="thermostatAutoMode" typeId="thermostatAutoMode">
+				<label>T&apos;stat Program Auto Mode</label>
+				<description>The thermostat is in Automatic Mode (Off = Manual Mode)</description>
+			</channel>
+
+			<channel id="occupancyModePresent"
+				typeId="occupancyModePresent">
+				<label>Occupancy Mode Present</label>
+				<description>The thermostat is in Present Occupancy Mode (Off = Away Mode)</description>
+			</channel>
+
+			<channel id="hotWaterAutoMode" typeId="hotWaterAutoMode">
+				<label>Hotwater Auto Mode</label>
+				<description>The domestic water heating is in Automatic Mode (Off = Manual Mode)</description>
+			</channel>
+
+			<channel id="hotWaterSwitchState"
+				typeId="hotWaterSwitchState">
+				<label>Hotwater Switch State</label>
+				<description>The On/Off state of the domestic water heating</description>
+			</channel>
+
+		</channels>
+
+		<properties>
+			<property name="vendor">Siemens</property>
+			<property name="modelId">RDS</property>
+		</properties>
+
+		<config-description>
+			<parameter name="plantId" type="text" required="true">
+				<label>Plant Id</label>
+				<description>The Plant Id of the thermostat in the Siemens Climatix IC cloud account</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Measured temperature value</description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+
+	<channel-type id="targetTemperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Target Temperature</label>
+		<description>Target temperature setting</description>
+		<state readOnly="false" pattern="%.1f %unit%" min="5" max="30"
+			step="0.5" />
+	</channel-type>
+
+	<channel-type id="thermostatOutputState">
+		<item-type>String</item-type>
+		<label>Heating or Cooling</label>
+		<description>The thermostat is calling for Heating, Cooling, or Neither</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="roomHumidity">
+		<item-type>Number</item-type>
+		<label>Humidity</label>
+		<description>Measured humidity value</description>
+		<state readOnly="true" pattern="%.0f %%r.H." />
+	</channel-type>
+
+	<channel-type id="roomAirQuality">
+		<item-type>String</item-type>
+		<label>Air Quality</label>
+		<description>Room Air Quality</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="greenLeafScore">
+		<item-type>String</item-type>
+		<label>Energy Savings Level </label>
```

Comment:
```suggestion
		<label>Energy Savings Level</label>
```

Reply:
Ok. I modified the description and the channel too.